src/ecdsa/mod.rs

```diff
@@ -269,6 +272,13 @@ impl Signature {
         }
         ret
     }
+
+    /// Verifies an ECDSA signature for `msg` using `pk` and the global [`SECP256K1`] context.
+    #[cfg(feature = "global-context")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "global-context")))]
+    pub fn verify_ecdsa(&self, msg: &Message, pk: &PublicKey) -> Result<(), Error> {
```

Comment:
This already exists on the `ecdsa::Signature` type so I'd say repeating `_ecdsa` here is redundant.

Reply:
I named it to be the same as the other `verify_` methods, with that in mind do you think its still redundant?

Reply:
Well, the other APIs live on `Secp256k1` so just `verify` would be ambiguous.

If we call this on an `ecdsa::Signature`, `verify` is clearly defined, isn't it? We are not suddenly going to take the bytes out if it and run the schnorr signature verification algorithm on it :)

Reply:
On `Secp256k1`, the functions need to include the signature scheme, otherwise we couldn't have multiple of them.

Reply:
Sounds fair, I'll change them. Cheers